diffpy/snmf/factorizers.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+import scipy.optimize
+
+
+def lsqnonneg(stretched_component_matrix, target_signal):
+    """Finds the weights of stretched component signals under one-sided constraint.
```

Comment:
see my note.  Please check that this is changed to it finding stretching factor and not weights.

Reply:
I apologize, the function is used to find weights, not stretching factors.